app/views/coronavirus_landing_page/components/landing_page/_stay-home-logo.html.erb

```diff
@@ -0,0 +1,48 @@
+    <path d="M89.3462 22.4265L80.5195 11.8934V32.9557L89.3462 22.4265Z" fill="#FFF500"/>
+    <path d="M249.583 4.06624C248.545 3.53865 247.082 3.11657 245.911 3.11657C244.501 3.11657 243.436 3.74969 243.436 5.12143C243.436 8.47542 250.758 6.91525 250.758 13.4612C250.758 16.9471 247.964 19.0574 244.261 19.0574C242 19.0574 240.053 18.5299 239.255 18.2924L239.494 14.701C240.798 15.2022 241.942 15.8354 243.569 15.8354C244.953 15.8354 246.287 15.1495 246.287 13.6458C246.287 10.0808 238.966 11.8256 238.966 5.25333C238.966 4.69935 239.179 0 245.409 0C247.112 0 248.177 0.290177 249.777 0.633113L249.583 4.06624Z" fill="white"/>
+    <path d="M263.26 14.6219H257.429L256.152 18.7409H251.944L258.121 0.316528H262.914L268.879 18.7409H264.405L263.26 14.6219ZM260.489 3.85518H260.436L258.227 11.5091H262.409L260.489 3.85518Z" fill="white"/>
+    <path d="M267.613 0.316528H272.3L275.577 13.7513H275.63L278.986 0.316528H283.461L277.838 18.7409H273.151L267.613 0.316528Z" fill="white"/>
+    <path d="M285.973 0.316528H296.704V3.53485H290.181V7.65385H296.385V10.8759H290.181V15.5225H297.024V18.7409H285.973V0.316528Z" fill="white"/>
+    <path d="M237.902 26.2478H242.269V41.3445H248.42V44.6721H237.902V26.2478Z" fill="white"/>
+    <path d="M250.72 26.2478H255.087V44.6721H250.72V26.2478Z" fill="white"/>
+    <path d="M257.216 26.2478H261.903L265.18 39.6826H265.233L268.59 26.2478H273.064L267.445 44.6721H262.758L257.216 26.2478Z" fill="white"/>
+    <path d="M275.211 26.2478H285.943V29.4661H279.419V33.5851H285.623V36.8072H279.419V41.4538H286.262V44.6721H275.211V26.2478Z" fill="white"/>
+    <path d="M298.829 29.9937C297.792 29.4661 296.328 29.0441 295.153 29.0441C293.743 29.0441 292.679 29.6772 292.679 31.0489C292.679 34.4029 300 32.8427 300 39.3887C300 42.8746 297.206 44.9849 293.504 44.9849C291.242 44.9849 289.296 44.4573 288.497 44.2199L288.737 40.6285C290.041 41.1297 291.185 41.7628 292.812 41.7628C294.195 41.7628 295.526 41.077 295.526 39.5733C295.526 36.0083 288.205 37.7531 288.205 31.1808C288.205 30.6268 288.417 25.9275 294.648 25.9275C296.351 25.9275 297.415 26.2177 299.016 26.5606L298.829 29.9937Z" fill="white"/>
+    <path d="M223.609 22.4265L214.778 11.8934V32.9557L223.609 22.4265Z" fill="#FFF500"/>
+  </g>
+  <defs>
+    <clipPath id="clip0">
+      <rect width="300" height="45" fill="white"/>
+    </clipPath>
+  </defs>
+  <%= tag.image(src: image_url("stay-home-logo.png"),
+    class: "covid__logos",
+    alt: alt_text,
+    "xlink:href": "") %>
```

Comment:
What's the purpose of this png? I notice the previous SVG's we have didn't have them.

Reply:
This is a fall back for browsers that do not support SVG like IE8

Reply:
Do we even support IE8? The page already looks heavily broken there so adding support for IE8 here, while having the  rest of it be broken seems like inconsistent, low value and complex.

It'd be great if we had some guidelines we followed on what browsers we support as it seems rather inconsistent where we (GOV.UK collectively) add fallbacks - seeming to fall to dev preferences than to guidelines. Service manual points to IE11 but this is a long way behind that.

Reply:
We don't support IE8 and yes I noticed a lot of the things broken when testing this.

I agree with regarding inconsistency, I will bring this up in our next meeting and maybe put in some guidelines on what should be supported and should be ignored.

SVG support seems to be fairly good so might be worth dropping this:
https://caniuse.com/svg